apps/judicial-system/digital-mailbox-api/src/app/modules/cases/case.service.ts

```
@@ -95,6 +96,13 @@ export class CaseService {
     lang?: string,
   ): Promise<CaseResponse> {
     const response = await this.fetchCase(id, nationalId)
+    const defendant = response.defendants[0]
+    const subpoenas = defendant.subpoenas
+
+    if (subpoenas && subpoenas[0].subpoenaId) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)